superset-frontend/src/dashboard/components/nativeFilters/FiltersConfigModal/FilterTabs.tsx

```diff
@@ -111,36 +71,42 @@ const FilterTabsContainer = styled(LineEditableTabs)`
     .ant-tabs-nav-list {
       overflow-x: hidden;
       overflow-y: auto;
+      width: 270px;
       padding-top: ${theme.gridUnit * 2}px;
       padding-right: ${theme.gridUnit}px;
       padding-bottom: ${theme.gridUnit * 3}px;
       padding-left: ${theme.gridUnit * 3}px;
-      width: 270px;
     }
 
     // extra selector specificity:
     &.ant-tabs-card > .ant-tabs-nav .ant-tabs-tab {
-      min-width: ${FILTER_WIDTH}px;
-      margin: 0 ${theme.gridUnit * 2}px 0 0;
-      padding: ${theme.gridUnit}px
-        ${theme.gridUnit * 2}px;
-      &:hover,
+      margin: 0;
+      padding: 0;
       &-active {
         color: ${theme.colors.grayscale.dark1};
-        border-radius: ${theme.borderRadius}px;
+        margin-bottom: ${theme.gridUnit / 2}px;
         background-color: ${theme.colors.secondary.light4};
 
         .ant-tabs-tab-remove > span {
           color: ${theme.colors.grayscale.base};
           transition: all 0.3s;
         }
+        .anticon {
+          color: ${theme.colors.grayscale.base};
+        }
+        :hover{
+          span {
+            color: ${theme.colors.grayscale.base} !important;
```

Comment:
We try to always avoid `!important`. Can you find another way to override the color? You can increase the specificity using `&`.